### ISSUE TITLE
Find new home for Presidential Campaign Finance Summaries: Current and Historical

### ISSUE THREAD

**Issue by AmyKort** [fecgov/fec-cms#2771]
We need to find a home on the new website for this presidential campaign finance summaries table. One idea is to add it to the information available at https://transition.fec.gov/press/campaign_finance_statistics.shtml#

Pres map: "Presidential Campaign Finance Summaries: Current and Historical" https://classic.fec.gov/press/bkgnd/pres_cf/pres_cf_Even.shtml

**Completion criteria:**
- [x] Make the page in wagtail to copy over the table from the classic page
- [x] Ensure links point to transition site
- [x] Have content team and/or stakeholders review before publishing
- [x] Once published, add links in the presidential public funding page's resource section and on the transition page.

**Comment by dorothyyeager** [comment]
That transition page is very tricky to edit. Looking at the classic page, we could do a custom page in the Campaign finance data section. That particular template allows for a "custom table" which might be a good fit for the table on the classic page. It could be either front end or content team work (if not hard-coded).

![image](https://user-images.githubusercontent.com/24437369/55172312-40ecb900-5150-11e9-8c35-1066a61e3558.png)

The PDF and .xlsx files and their links should be moved into the CMS and redirected to get them off transition (where they are living now). That will be a bit of work because of the large amount of files. The content team could do it, but it might be quicker for the front end team to do it. 

Once the page is in order and live, a link to it needs to be added to the presidential public funding page... https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/#reports-and-resources

![image](https://user-images.githubusercontent.com/24437369/55172500-84472780-5150-11e9-9f6c-8d3bb1dbf899.png)

**Comment by kathycarothers** [comment]
I agree with Dorothy with creating a custom page in this section.

**Comment by dorothyyeager** [comment]
I have questions for @AmyKort @PaulClark2:

First, I notice that the [transition page](https://transition.fec.gov/press/campaign_finance_statistics.shtml#) has activity up through 12/31/2018 (see 
![image](https://user-images.githubusercontent.com/24437369/55559479-4ac86c00-56bc-11e9-9d63-a4d2f9374249.png)

And the classic page has activity through 2016.

When I create the new table, should I make sure there are rows up through 12/31/2018? 

Second question, to create the table in a way that makes links to PDF and Excel files (like the classic page), it will need to use HTML. Will Press be comfortable with making future updates to the page (if it's desired to do that)?

**Comment by dorothyyeager** [comment]
Turns out the dense table format from the pattern library might work best for such a complicated table. Still requires HTML in order to code the links within the table. Documenting that here since we really need a Wagtail table that allows a user not comfortable with html coding to make a link.

**Comment by AmyKort** [comment]
I defer to @PaulClark2 and his better judgment on these types of issues.

**Comment by PaulClark2** [comment]
@dorothyyeager the new table should I make sure there are rows up through 12/31/2018. I think we can teach Carmen, Myles and Christian how to insert the HTML tags. Perhaps a new page template could be a future Wagtail enhancement.

**Comment by dorothyyeager** [comment]
Thanks! Definitely need that kind of table block in Wagtail one day since we are using tables with links in them for dates and deadlines and trainings pages too. 

In meantime, I'll go ahead and prepare this page through 2018. I can also add the blocks for 2019-2020 that can be populated this summer and beyond with links once available. Will ping in Slack when this is ready for review.

**Comment by dorothyyeager** [comment]
Chatted with @PaulClark2 some more, and since we are talking about the classic page, we only need it to go through 2015-2016. The draft page has been made and is ready for review.

**Comment by dorothyyeager** [comment]
We did a review during Design Sync and team members offered good feedback on the visual appearance issues such a giant table presents. While we can't do all the suggestions now, due to the urgency of getting this moved into the CMS, I'm going to try splitting up the election cycles into separate tables and limiting the word "reports" in the heading.

**Comment by dorothyyeager** [comment]
I've made the fixes discussed in Design Sync as well as added some code suggested by @johnnyporkchops and the appearance is much improved. I've asked content/design team to take a second look.

**Comment by dorothyyeager** [comment]
Thank you @PaulClark2 and @JonellaCulmer for looking it over. Paul advises Press does not need to review it, so I've published the page at https://www.fec.gov/campaign-finance-data/presidential-campaign-finance-summaries/. I've also put in a www redirect for the old classic page. It was not linked on the transition page to begin with, so I left that alone. I'll add redirecting the classic URL to our redirect spreadsheet.

I've added the link to the new page and to the related transition page at https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/#reports-and-resources 
![image](https://user-images.githubusercontent.com/24437369/55729921-0a7f2b80-59e5-11e9-96af-a4e0f3409c9f.png)

**Future work**
-I'm not able to edit the classic page to add a link, so that will need to be done. 
-When we're able to make a table and retain the template with the narrower column for text plus a read next box, we'll want to revisit this page.

For now, with the page up, we are ready to close this issue.
